games/z.yaml

```diff
@@ -140,6 +140,21 @@
   - 'https://www.zeldaclassic.com/wp-content/uploads/2016/12/ss_hero_dreams.png'
   - 'https://www.zeldaclassic.com/wp-content/uploads/2016/12/ss_quest744.png'
 
+- name: 'The Legend of Zelda: Ocarina of Time'
+  originals:
+  - 'The Legend of Zelda: Ocarina of Time'
+  type: remake
+  repo: 'https://github.com/zeldaret/oot'
+  url: 'https://zelda64.dev/'
+  development: active
+  status: playable
+  license:
+  - Custom
```

Comment:
should this be As-is? I can't find a license in the repo

Reply:
Yes, it is just like re3 without a license.